src/main/java/com/example/Restaurant_Management_System_REST_API/configuration/SecurityConfig.java

```diff
@@ -41,6 +41,10 @@ public SecurityFilterChain securityFilterChain(HttpSecurity httpSecurity, MvcReq
                         .requestMatchers(mvc.pattern("/api/customer/find/**"), mvc.pattern("/api/customer/findAll"))
                         .hasAnyRole("OWNER", "MANAGER", "STAFF")
 
+                        //Secured paths for Inventory item operations
+                        .requestMatchers(mvc.pattern("/api/inventory/**"))
+                        .hasAnyRole("OWNER", "MANAGER", "STAFF")
+
```

Comment:
Great job on keeping this code base clean! Also a plus for having clean naming convention and distinguished roles.